uamqp/client.py

```diff
@@ -81,7 +81,7 @@ class AMQPClient(object):
     :type max_frame_size: int
     :param channel_max: Maximum number of Session channels in the Connection.
     :type channel_max: int
-    :param idle_timeout: Timeout in milliseconds after which the Connection will close
+    :param idle_timeout: Timeout in seconds after which the Connection will close
```

Comment:
This PR seems very small - we only needed to fix doc strings? We didn't need to add any multipliers to the values?

Reply:
I double check the code and find it's just doc strings.
It must be in the event hub code where I keep flipping between milliseconds and seconds that makes me feel dizzy 🤣